recipes-bsp/libubootenv/libubootenv/imx7s-warp/fw_env.config

```diff
@@ -1,5 +1,5 @@
 # WaRP7 i.MX7
 
 # Block device name	Device offset	Env. size
-/dev/mmcblk1 0x80000 0x2000
-/dev/mmcblk1 0xA0000 0x2000
+/dev/mmcblk1 0xC0000 0x2000
```

Comment:
Can you elaborate the commit log ?

Maybe a reference to this commit [1] is sufficient ;)

[1] - https://gitlab.denx.de/u-boot/u-boot/-/commit/1fae23899c7d6a7f44dec6a963d6c74ce5fe5c19